src/components/common/Dialogs/AwardAndSmiteDialogs/ManageReputationDialogForm/ManageReputationDialogForm.tsx

```diff
@@ -179,7 +195,7 @@ const ManageReputationDialogForm = ({
     }
   }, [schemaUserReputation, isSmiteAction, trigger]);
 
-  const formattedData = verifiedUsers.map((user) => ({
+  const formattedData = users.map((user) => ({
     ...user,
     id: user,
```

Comment:
Not your changes, but is that right? User object as an id instead of just the address?

Reply:
One for @ArmandoGraterol

Reply:
That part of the code ain't mine :P

Nevertheless, I think it's wrong, the `id` should be the address instead of the whole object